virus-interpreter/src/main/java/com/hartwig/hmftools/virusinterpreter/algo/VirusBlacklistingDb.java

```diff
@@ -0,0 +1,26 @@
+package com.hartwig.hmftools.virusinterpreter.algo;
+
+import com.hartwig.hmftools.common.virus.TaxidType;
+
+import org.immutables.value.Value;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Value.Immutable
+@Value.Style(allParameters = true,
+             passAnnotations = { NotNull.class, Nullable.class })
+public abstract class VirusBlacklistingDb
+{
+
+    @NotNull
+    public abstract Integer taxid();
```

Comment:
This should be `int` :)

(your favourite kotlin vs java issue)

Reply:
I will put this above my bed, so I can't forget anymore :)